wagtailio/static/sass/abstracts/_mixins.scss

```diff
@@ -94,6 +94,14 @@
     }
 }
 
+// Places the focus indicator within the element
+@mixin focus-internal() {
+    &:focus-visible {
+        outline-offset: -4px;
+        outline: 2px solid currentColor;
```

Comment:
```suggestion
```

It seems to me that we could move the focus indicator within, without introducing new styles for it.